pkg/cmd/ctlstore/main.go

```diff
@@ -470,6 +482,82 @@ func reflector(ctx context.Context, args []string) {
 	reflector.Start(ctx)
 }
 
+func multiReflector(ctx context.Context, args []string) {
```

Comment:
I don't love stuffing all this in the cmd/main package. If this becomes burdensome to maintain and test, I can move it to an `multi_orchestrate.go` package in the `reflector` package.